app/controllers/devise/sessions_controller.rb

```diff
@@ -0,0 +1,74 @@
+class Devise::SessionsController < DeviseController
+    prepend_before_action :require_no_authentication, only: [:new, :create]
+    prepend_before_action :allow_params_authentication!, only: :create
+    prepend_before_action :verify_signed_out_user, only: :destroy
+    prepend_before_action(only: [:create, :destroy]) { request.env["devise.skip_timeout"] = true }
+  
+    def new
+      self.resource = resource_class.new(sign_in_params)
+      clean_up_passwords(resource)
+      yield resource if block_given?
+      respond_with(resource, serialize_options(resource))
+    end
+  
+    def create
+      if warden.authenticate
+        self.resource = warden.authenticate!(auth_options)
+        set_flash_message!(:notice, :signed_in)
+        sign_in(resource_name, resource)
+        yield resource if block_given?
+        respond_with resource, location: after_sign_in_path_for(resource)
+      else
+        render json: {errors: "Invalid Email or password."}
+      end
+    end
+  
+    def destroy
+      signed_out = (Devise.sign_out_all_scopes ? sign_out : sign_out(resource_name))
+      set_flash_message! :notice, :signed_out if signed_out
+      yield if block_given?
+      # respond_to_on_destroy
```

Comment:
could you remove this commented out line? :)

Reply:
Done